src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardPlayerRequestResolver.java

```diff
@@ -119,7 +121,7 @@ public IRequest getFollowupRequestForCompletion(@NotNull final IRequestManager m
     @Override
     public IRequest onRequestCancelledOrOverruled(@NotNull final IRequestManager manager, @NotNull final IRequest request) throws IllegalArgumentException
     {
-        return null;
+        return getFollowupRequestForCompletion(manager, request);
```

Comment:
@Raycoms This is what was causing the bug. When i reassign i cancel the request. However the SPRR was not updating its internal dataset. The error only happened when and only when the DMan picked up a stack that was FINALLY assigned to the Player, since the SRRR does handle reassignments properly as it makes use of that functionality.